smart-contracts/package.json

```diff
@@ -33,7 +30,10 @@
     "eslint-plugin-standard": "^4.0.0",
     "remix-ide": "^0.7.5",
     "remixd": "^0.1.8-alpha.6",
-    "solhint": "^1.4.0"
+    "shelljs": "^0.8.3",
+    "solhint": "^1.4.0",
+    "zos": "^v2.1.0-rc.0",
```

Comment:
Zos is not required to build, but will be required for automated deploymen, so I'm not sure where that puts it in the current CI setup...

Reply:
For now, let's keep our out and add it back when we automate deployments..